### ISSUE TITLE
Moodle 4.4 and 4.5 support

### ISSUE THREAD

**Issue by mojaedukacja** [catalyst/moodle-tool_mfa#462]
Whet MFA will work with Moodle 4.4 or 4.5? 

Every time I try to update moodle with MFA I get warring/errors like that
![obraz](https://github.com/user-attachments/assets/a06cd1e4-5992-44e1-addd-03fbedd2a014)

if I update to server this two files.... moodle stop working after update.

**Comment by ArslanKathia** [comment]
Moodle 4.4 and 4.5 now include Multi-Factor Authentication (MFA) as a core feature. If you're already using an MFA plugin in a previous Moodle version, it's recommended to disable that plugin before upgrading to the new version.

**Comment by danmarsden** [comment]
yes - please do not overwrite your Moodle core files in moodle 4.5 and higher with the plugin avaialble here - you should just use the core moodle version of this tool instead.
